public/app/features/plugins/admin/components/PluginDetailsBody.tsx

```diff
@@ -52,30 +70,14 @@ export function PluginDetailsBody({ plugin, queryParams, pageId }: Props): JSX.E
 
   // Permissions will be returned in the iam field for installed plugins and in the details.iam field when fetching details from gcom
   const permissions = plugin.iam?.permissions || plugin.details?.iam?.permissions;
-  if (config.featureToggles.externalServiceAccounts && pageId === PluginTabIds.IAM) {
```

Comment:
Should this feature toggle and the page condition still be used?

Reply:
Changed as requested.